app/cdap/components/GridTable/types.ts

```diff
@@ -42,9 +42,22 @@ export interface IExecuteAPIResponse {
 export interface IHeaderNamesList {
   name: string;
   label: string;
-  type: Array<string | number | IRecords | boolean>;
+  type: string[];
```

Comment:
plz make sure there arent any errors in terminal, this might cause an error, if there are any terminal errors, plz fix them